pkg/fleetautoscalers/fleetautoscalers.go

```diff
@@ -221,18 +221,19 @@ func applyBufferPolicy(b *autoscalingv1.BufferPolicy, f *agonesv1.Fleet) (int32,
 		replicas = int32(math.Ceil(float64(f.Status.AllocatedReplicas*100) / float64(100-bufferPercent)))
 	}
 
-	limited := false
+	scalingDownLimited := false
+	scalingUpLimited := false
 
 	if replicas < b.MinReplicas {
 		replicas = b.MinReplicas
-		limited = true
+		scalingDownLimited = true
```

Comment:
Sorry for the back and force but better to change this and line 233, 236 too for consistency